jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SlideGroupsTests.java

```diff
@@ -31,109 +27,74 @@ public void before() {
 
     @Test
     public void activeClassSlideGroupTests() {
+        activeClassSlideGroup.startTest();
         activeClassSlideGroup.is().displayed();
-        testCommonSlideGroup(activeClassSlideGroup);
+
+        activeClassSlideGroup.clickOnSlideByIndex(1);
+        activeClassSlideGroup.has().slideSelected(1);
+        activeClassSlideGroup.clickOnSlideByIndex(2);
+        activeClassSlideGroup.has().slideSelected(2);
+        activeClassSlideGroup.has().slideNotSelected(1);
+
+        if (activeClassSlideGroup.nextButtonIsActive()) {
```

Comment:
don't use if logic into tests. You know test elements structure and can use it into the test.